api/src/main/java/org/pmiops/workbench/access/AccessModuleService.java

```diff
@@ -3,13 +3,18 @@
 import java.sql.Timestamp;
 import java.util.List;
 import java.util.Optional;
+import org.pmiops.workbench.config.ConfigConstants;
 import org.pmiops.workbench.db.model.DbAccessModule.DbAccessModuleName;
 import org.pmiops.workbench.db.model.DbUser;
 import org.pmiops.workbench.model.AccessBypassRequest;
 import org.pmiops.workbench.model.AccessModule;
 import org.pmiops.workbench.model.AccessModuleStatus;
 
 public interface AccessModuleService {
+  static boolean isSignedDuccVersionCurrent(Integer signedVersion) {
```

Comment:
can move here because it no longer depends on instance data

Reply:
For `static` methods where we use the Service/Impl pattern, we may not want to go to the trouble of constructing an impl - they often have many dependencies.  The typical use case is something simple like this, where I'd be inclined to use `static` on a class as well.

Interfaces also have `default` methods which work differently.  You can think of these as instance methods on every implementing class.  We often use `default` in Daos and Mappers, because we don't have access to the implementing classes: the MapperImpls are generated, and the Dao impls are so auto-magic that we never see them at all.